web/annotation_layer_builder.js

```diff
@@ -60,9 +60,11 @@ class AnnotationLayerBuilder {
   /**
    * @param {PageViewport} viewport
    * @param {string} intent (default value is 'display')
+   * @returns {Promise} A promise resolved when rendering the annotations is
+   * complete.
```

Comment:
We standardized the JSDoc comments recently to have two-space indentation if the line belongs to a previous line, so `complete` should have two extra spaces before it.